docs/_docs/04_reference/built-in-functions.md

```diff
@@ -610,3 +612,7 @@ inputs = merge(
   }
 )
 ```
+
+## get\_terragrunt\_source
+
+`get_terragrunt_source()` returns the location of the Terraform files. This location can be provided via CLI `--terragrunt-source`, an environment variable `TERRAGRUNT_SOURCE` or in the `terragrunt.hcl` file. Calling this function will return the value of that field.
```

Comment:
```suggestion
`get_terragrunt_source()` returns the value passed in via the CLI `--terragrunt-source` or an environment variable `TERRAGRUNT_SOURCE`. Note that this will return an empty string when either of those values are not provided.

This is useful for constructing before and after hooks, or TF flags that only apply to local development (e.g., setting up debug flags, or adjusting the `iam_role` parameter).
```